databricks_cli/sdk/api_client.py

```diff
@@ -62,16 +62,8 @@ class ApiClient(object):
     A partial Python implementation of dbc rest api
     to be used by different versions of the client.
     """
-    def __init__(self, user = None, password = None, host = None, token = None, configUrl = None,
-            apiVersion = version.API_VERSION, default_headers = {}, verify = True):
-        if configUrl:
-            self.url = configUrl
-            params = self.performQuery("/", headers = {})[1]
-            params = credential.json
-            user = str(params["user"])
-            password = str(params["password"])
-            host = str(params["apiUrl"].split("/api")[0])
-
+    def __init__(self, user = None, password = None, host = None, token = None,
+                 apiVersion = version.API_VERSION, default_headers = {}, verify = True):
```

Comment:
Not part of this PR but `{}` shouldn't be the default value because it is immutable.